kong/conf_loader.lua

```diff
@@ -70,6 +70,7 @@ local CONF_INFERENCES = {
   latency_tokens = {typ = "boolean"},
   error_default_type = {enum = {"application/json", "application/xml",
                                 "text/html", "text/plain"}},
+  client_max_body_size = {typ = "string"},
```

Comment:
Yes, this is indeed for explicitness. All `string` values are still listed here.